### PR TITLE
Add "Fine-tune ALBERT for sentence-pair classification" notebook to the community notebooks

### PR DESCRIPTION
**Pull request by NadirEM**
Hello,

I'm adding to the community notebooks a tutorial on fine-tuning ALBERT and other BERT-based models for sentence-pair classification.
The main features of this tutorial are :
[1] End-to-end ML implementation (training, validation, prediction, evaluation)
[2] Easy adaptability to your own datasets
[3] Facilitation of quick experiments with other BERT-based models (BERT, ALBERT, ...)
[4] Quick training with limited computational resources (mixed-precision, gradient accumulation, ...)
[5] Multi-GPU execution
[6] Threshold choice for the classification decision (not necessarily 0.5)
[7] Freeze BERT layers and only update the classification layer weights or update all the weights
[8] Reproducible results with seed settings

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Very nice, thank you for your contribution!
